spec/moleculer/serializers/serializer_behavior.rb

```diff
@@ -9,9 +9,14 @@
 
 RSpec.shared_examples "deserialize packet" do |data, packet|
   it "should deserialize the #{packet.name} class" do
-    serialized_packet = subject.serialize(packet.new(config, data))
+    original_packet     = packet.new(config, data)
+    serialized_packet   = subject.serialize(original_packet)
+
     expect { subject.deserialize(packet.name.split("::").last.downcase.to_sym, serialized_packet) }.to_not raise_error
     expect(subject.deserialize(packet.name.split("::").last.downcase.to_sym, serialized_packet)).to be_a packet
+
+    deserialized_packet = subject.deserialize(packet.name.split("::").last.downcase.to_sym, serialized_packet)
+    expect(deserialized_packet).to eq(original_packet)
```

Comment:
This could be considered multiple tests. An RSpec context might make it easier to change it to that style. However, his is more of a preference thing, to be honest. The test itself is fine, just a little heavy for my liking.